src/DynamoCore/Graph/Connectors/ConnectorModel.cs

```diff
@@ -215,29 +215,6 @@ protected override void SerializeCore(XmlElement element, SaveContext context)
         protected override void DeserializeCore(XmlElement nodeElement, SaveContext context)
```

Comment:
Since this fuction does not do anything now and is an override. Can we remove?

Reply:
@QilongTang 
Visual Studio is reporting that it has 56 references, then I guess is not a good idea to update 56 places in about 20 different files (probably).

![image](https://user-images.githubusercontent.com/61755417/86300581-eea70d00-bbc8-11ea-944a-d51618092991.png)

Reply:
@RobertGlobant20 Sure